fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AccountNumberPreferencesTest.java

```diff
@@ -376,7 +376,7 @@ private void validateAccountNumberLengthAndStartsWithPrefix(final String account
             Assert.assertEquals(accountNumber.length(), prefix.length() + 9);
             Assert.assertTrue(accountNumber.startsWith(prefix));
         } else {
-            Assert.assertEquals(accountNumber.length(), 9);
+            Assert.assertEquals( 9,accountNumber.length());
```

Comment:
Does this stuff also reject space separated commas. It's not a big deal I'd prefer if there is a space after the comma. Makes stuff more readable.

Reply:
Ok done that